output_chunks.cc

```diff
@@ -1012,6 +1012,7 @@ void DynsymSection<E>::copy_buf(Context<E> &ctx) {
     } else {
       esym.st_shndx = sym.input_section->output_section->shndx;
       esym.st_value = sym.get_addr(ctx, false);
+      esym.st_visibility = sym.esym().st_visibility;
```

Comment:
We merge symbol visibility when we see more than one symbol (this process is done in`ObjectFile::merge_visibility`). Merged visibility is accessible as `sym.visibility`. So please use `sym.visibility` instead of `sym.esym().st_visibility`.